### ISSUE TITLE
Populate Loki Helm Chart CHANGELOG.md with v5.0.0 Changes

### ISSUE THREAD

**Issue by seanocca** [grafana/loki#9122]
The [CHANGELOG.md](https://github.com/grafana/loki/blob/helm-loki-5.0.0/production/helm/loki/CHANGELOG.md) file in Loki Helm Chart v5.0.0 is missing the changes from v4.10.0 to v5.0.0. 

According to the [CHANGELOG.md](https://github.com/grafana/loki/blob/helm-loki-5.0.0/CHANGELOG.md) file in the main repository of the chart there have been quite a few updates to the deployment. However this is not verified in the Helm Chart [CHANGELOG.md](https://github.com/grafana/loki/blob/helm-loki-5.0.0/production/helm/loki/CHANGELOG.md).

Can someone please update this release (or at least the main branch) with the correct changes made from v4.10.0 to v5.0.0?

**Comment by jeschkies** [comment]
Yes. That was my bad. I let https://github.com/grafana/loki/commit/afee8bcabf5bba6c962eaad039eca4174a0a867a through without a changelog entry. It feels releasing a new version on each `main` build might be a little too much. Also, we may want to auto bump the patch version instead of relying on our contributors.

What do you think @trevorwhitney?

**Comment by mjnagel** [comment]
Related issue - https://github.com/grafana/loki/issues/9120

**Comment by trevorwhitney** [comment]
Yes, I agree. We need to come up with a release process for the helm chart.

I propose that we do a weekly release that bumps the minor version of the chart, and we only bump the major version of the chart when the underlying Loki/GEL version changes. Of course this breaks true semver, as there's no good way to determine if a weekly has breaking changes or not.

If we could abandon semver altogether I would, but helm requires a semver v2 version, so we're constrained by that.

If this is not tenable to the community, then I think we need to adopt a much slower release process, maybe one tied into the Loki release process. Either that, or we would someone from the community to volunteer to shepherd out more frequent releases.

Regardless, the current process is very broken, and we shouldn't be cutting a new release with every PR.

What does everyone think?

**Comment by hervenicol** [comment]
My 2 cents, FWIW:

1. Chart development is quite distinct from app dev, and releases / breaking changes can't always be synchronized.
    So I quite like the "one release per xx days when some changes were contributed".
    But I think weekly releases may be too much. I'd go for 2 weeks or even 1 month. 

2. You wouldn't be the first one to use "broken semver-like".
   Personally, when I upgrade an app, even for patch versions, I try to read the changelog and diff the code for helm charts.


3. Why not extending the list of keywords for the CHANGELOG with [BREAKING] or something similar, and base the versionning choices on the keywords?

4. Anyway the current problem is there's been some changes with no entry in the CHANGELOG. Can that be enforced, with CI maybe?

**Comment by JStickler** [comment]
Closing as Stale
As of March 16, 2026, the Loki Helm Chart has been forked to the new [Grafana-Community/helm-charts repo](https://github.com/grafana-community/helm-charts) and is now being maintained by the Grafana Community and Grafana Champions. If this issue affects Grafana Enterprise Logs (GEL), please leave a comment here.  Otherwise, if this is still an issue, please reopen your issue here [here](https://github.com/grafana-community/helm-charts/issues).
